src/api_clients/pocket_publisher_database_execute.py

```diff
@@ -0,0 +1,97 @@
+import os
+from typing import Any
+
+from prefect.tasks.mysql import MySQLExecute
+from prefect.utilities.tasks import defaults_from_attrs
+
+
+class PocketPublisherDatabaseExecute(MySQLExecute):
+    """
+    PocketPublisherDatabaseQuery automatically connects to our Publisher MySQL Database if the right environment variables are set:
+    - POCKET_PUBLISHER_DATABASE_HOST
+    - POCKET_PUBLISHER_DATABASE_PORT
+    - POCKET_PUBLISHER_DATABASE_DBNAME
+    - POCKET_PUBLISHER_DATABASE_USER
+    - POCKET_PUBLISHER_DATABASE_PASSWORD
```

Comment:
If you add these variables to [parameterStoreNames](https://github.com/Pocket/data-flows/blob/main/.aws/src/config/index.ts#L35-L37) and create corresponding SSM Parameters, they will be injected in the environment.